src/js/scope_events.js

```diff
@@ -195,7 +195,7 @@
 		fireEvent('change', handleNumber);
 
 		if ( options.events.snap ) {
-			start(event, { handles: [scope_Handles[total]] });
+			start(event, { handles: [scope_Handles[handleNumber]] });
```

Comment:
Awesome work you are filling the gaps in my PRs, hope it's ok. We will make an awesome module! Also I have this already working: https://github.com/algolia/react-nouislider